garageScript/c0d3-app
# Make AdminLessonNav more generic

Closes https://github.com/garageScript/c0d3-app/issues/2268

This pull request changes:
* AdminLessonNav is now more generic and called NavCard
* NavCard takes in more simplified props
* Makes it easier for the DOJO exercises page to use this NavCard component (https://github.com/garageScript/c0d3-app/pull/2265)

How to test:
* Go to /admin/lessons/js1/modules
* Verify that NavCard works and looks exactly how it looked and worked before

![navcard](https://user-images.githubusercontent.com/7637655/189109294-859ea731-576b-4fa2-b840-2f0e6b1f6488.png)
